lxd/storage/backend_lxd.go

```diff
@@ -1912,7 +1912,7 @@ func (b *lxdBackend) recvBlockVol(toFile *os.File, volName string, conn io.ReadW
 
 	var wrapper *ioprogress.ProgressTracker
 	if args.TrackProgress {
-		wrapper = migration.ProgressTracker(op, "block_progress", volName)
+		wrapper = migration.ProgressTracker(op, "block_progress", "Transferring instance")
```

Comment:
Are these ever used with non instance volumes?

Reply:
Currently, those are used only within conversion API. Therefore, they are used only for instance volume transfer